custom-template/package.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
found the other way to inline svgs (through Markup template of node-kss), will remove this